rule.jison

```diff
@@ -40,6 +40,7 @@
 [0-9]+("."[0-9]+)?\b  return 'NUMBER'
 \"[^\"]*\"            return 'STRING'
 [a-z]+                return 'VARIABLE'
+\[.*?\]               return 'ARRAY'
```

Comment:
It might not be a good idea to tokenize array like this. It shall be parsed by grammar to take care of a lot of cases. What if someone write an array like this: operators in [ "(", ")", "[", "]", "+" ]?